src/platform/mbed/net/src/net_if.cpp

```diff
@@ -5,6 +5,10 @@
 #include <ifaddrs.h>
 #include <sys/ioctl.h>
 
+#include "mbed-trace/mbed_trace.h"
+
+#define TRACE_GROUP "NETS"
```

Comment:
Maybe it would be better to change the trace group to "NETI" to distinguish it from the net socket module